docs/src/pages/components/toggle-button/ToggleButtonSizes.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import FormatAlignLeftIcon from '@material-ui/icons/FormatAlignLeft';
+import FormatAlignCenterIcon from '@material-ui/icons/FormatAlignCenter';
+import FormatAlignRightIcon from '@material-ui/icons/FormatAlignRight';
+import FormatAlignJustifyIcon from '@material-ui/icons/FormatAlignJustify';
+import Grid from '@material-ui/core/Grid';
+import ToggleButton from '@material-ui/lab/ToggleButton';
+import ToggleButtonGroup from '@material-ui/lab/ToggleButtonGroup';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import Radio from '@material-ui/core/Radio';
+
+const styles = theme => ({
+  toggleContainer: {
+    margin: theme.spacing(4, 0),
+  },
+  sizePicker: {
+    margin: theme.spacing(2, 0),
+  },
+});
+
+class ToggleButtonSizes extends React.Component {
+  state = {
+    alignment: 'left',
+    size: 'medium',
+  };
+
+  handleAlignment = (event, alignment) => {
+    this.setState({ alignment });
+  };
+
+  handleSizeChange = key => (event, value) => {
+    this.setState({
+      [key]: value,
+    });
+  };
+
+  render() {
+    const { classes } = this.props;
+    const { alignment, size } = this.state;
+
+    return (
+      <Grid container spacing={2}>
```

Comment:
Could we set the height so that size doesn't jump when changing the ToggleButons size?